digits/config/store_option.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2015-2016, NVIDIA CORPORATION.  All rights reserved.
+from __future__ import absolute_import
+
+from . import config_option
+from . import prompt
+
+class ModelStoreOption(config_option.Option):
+
+    @staticmethod
+    def config_file_key():
+        return 'model_store'
+
+    @classmethod
+    def prompt_title(cls):
+        return 'Model Store URL list'
+
+    @classmethod
+    def visibility(cls):
+        return config_option.Visibility.HIDDEN
```

Comment:
Can you make this `DEFAULT` instead? Otherwise I have to use `python -m digits.config.edit -v` to edit it.